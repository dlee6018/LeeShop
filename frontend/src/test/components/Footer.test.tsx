import {shallow} from "enzyme"
import React from 'react'
import Footer from '../../components/Footer'

it('render a footer component', () => {
    expect(shallow(<Footer/>)).toMatchSnapshot()
});
