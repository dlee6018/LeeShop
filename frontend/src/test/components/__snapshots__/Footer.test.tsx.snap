// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render a footer component 1`] = `
<footer>
  <Container
    fluid={false}
  >
    <Row
      noGutters={false}
    >
      <Col
        className="text-center py-3"
      >
        Copyright Â© LeeShop
      </Col>
    </Row>
  </Container>
</footer>
`;
